js/tests/unit/tooltip.js

```diff
@@ -389,17 +392,24 @@ $(function () {
     $topTooltip.bootstrapTooltip('hide')
     equal($('.tooltip').length, 0, 'top positioned tooltip removed from dom')
 
-    var $rightTooltip = $('<div style="right: 0;" rel="tooltip" title="Right tooltip">Right Dynamic Tooltip</div>')
+    var $rightTooltip = $('<a rel="tooltip" title="Right tooltip">Right Dynamic Tooltip</a>')
       .appendTo($container)
       .bootstrapTooltip({ placement: 'right auto' })
 
     $rightTooltip.bootstrapTooltip('show')
+    ok($('.tooltip').is('.right'), 'left positioned tooltip is positioned right')
```

Comment:
Might be clearer just to create your own tooltip instances rather than reusing `$rightTooltip` and `$leftTooltip`.